### PR TITLE
Creating a stub of a file for colchester

### PR DESCRIPTION
**Pull request by emorr11**
What would you like to know?

### CODE REVIEW

File: places/emorr11-colchester.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Colchester, IL
+
+### Beck's 
+
```
Comment (goba86) [inline places/emorr11-colchester.md]: Where is the best deep dish pizza?

Comment (emorr11) [inline places/emorr11-colchester.md]: Only 1 restaurant in town...no pizza :(

File: places/emorr11-colchester.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+### N/a
+
```
Comment (goba86) [inline places/emorr11-colchester.md]: Are there any fun bars in town?

Comment (emorr11) [inline places/emorr11-colchester.md]: No! It is a dry town

File: places/emorr11-colchester.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+### Argyle State Park
```
Comment (goba86) [inline places/emorr11-colchester.md]: Are there any bears in this park and do they like picnic baskets?

**Pull request merged**